internal/servicedeployer/custom_agent.go

```diff
@@ -70,7 +70,7 @@ func NewCustomAgentDeployer(options CustomAgentDeployerOptions) (*CustomAgentDep
 
 // SetUp sets up the service and returns any relevant information.
 func (d *CustomAgentDeployer) SetUp(ctx context.Context, svcInfo ServiceInfo) (DeployedService, error) {
-	logger.Debug("setting up service using Docker Compose service deployer")
+	logger.Debug("DEPRECATED - setting up service using Docker Compose service deployer")
```

Comment:
Could it be added something else?

Reply:
We should increase the log level, at least to be a warning.

Reply:
Sure, I forgot about the log level.
Thanks!